package.json

```diff
@@ -26,9 +26,9 @@
   },
   "homepage": "https://github.com/codeyourfuture/eslint-config-standard#readme",
   "devDependencies": {
-    "eslint": "^6.0.0"
+    "eslint": "^6 || ^7"
```

Comment:
I think it would make sense for the development dependency to be more specific. We can bump it to the latest version, though, so I'd expect to see `"^7.5.0"` here. If you do that using `npm install --save-dev eslint@7`, rather than editing the `package.json` directly, this should also create changes in `package-lock.json` that you can include in the commit.

Reply:
Thank you! I've installed the latest version for ESLint ("^7.5.0") using `npm install --save-dev...`,  and it has updated the package-lock.json file.